### PR TITLE
stylo: Remove calc support from media queries

### PR DESCRIPTION
**Pull request by canova**
Gecko currently doesn't support calc inside media queries. We should
also remove the calc support temporarily for parity with gecko. We can
add this support back in next releases.

Reviewed by emilio in Bugzilla.

---
- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix [Bug 1390339](https://bugzilla.mozilla.org/show_bug.cgi?id=1390339)

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/18349)

### CODE REVIEW

File: components/style/gecko/media_queries.rs [hunk 1]
```diff
@@ -555,10 +556,19 @@ impl Expression {
 
             let value = match feature.mValueType {
                 nsMediaFeature_ValueType::eLength => {
-                    MediaExpressionValue::Length(
-                        specified::Length::parse_non_negative(context, input)?)
+                    let length = Length::parse_non_negative(context, input)?;
+                    // FIXME(canaltinova): Gecko doesn't support calc inside media
+                    // queries. This check is for temporarily remove it for parity
+                    // with gecko. We should remove this check when we want to support it.
```
Comment (emilio) [inline components/style/gecko/media_queries.rs]: Please mention the bug you filed.

**Review by emilio — CHANGES_REQUESTED** [review]

**Comment by emilio** [comment]
@bors-servo r-

r=me, but mentioning the bug is necessary.

**Comment by jdm** [comment]
@bors-servo retry
* Build timed out

**Pull request merged**